authors/apps/articles/serializers.py

```diff
@@ -14,16 +16,8 @@ def to_representation(self, data):
         '''
         Show author's actual details instead of author's id
         '''
-        article_details = super(ArticleSerializer,
-                                self).to_representation(data)
-        if User.objects.filter(pk=int(article_details["author"])).exists():
-            user_details = User.objects.get(pk=int(article_details["author"]))
-            article_details["author"] = {
-                "id": user_details.id,
-                "username": user_details.username,
-                "email": user_details.email}
-            return article_details
-        raise APIException({"error": "User does not exist"})
+        article_details = super().to_representation(data)
```

Comment:
I like the refactoring you did for `to_representation`. It prevents repetitions

Reply:
Thank you